andrew749/letstalk
# Add UW email verification

Server:
- add `POST /send_email_verification` to send a new verification email
- add `POST /verify_email` callback to verify a UW email
- add Verify Email template to SendGrid
- add new `users.is_email_verified` column, `verify_email_ids` table
- add new user bootstrap state `account_email_verified`

App:
- add VerifyEmailView (after signup, before onboarding) to input a UW email address

Landing:
- add `/verify_email?requestId=` endpoint, static page linked to from email
